sgl-kernel/csrc/cpu/gemm.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <ATen/native/CPUBlas.h>
+#include "common.h"
```

Comment:
```suggestion
```

Reply:
Need this due to the build failure `In file included from /home/yanbingj/project/sglang/sgl-kernel/csrc/cpu/gemm.cpp:1:
/home/yanbingj/project/sglang/sgl-kernel/csrc/cpu/gemm.h:55:5: error: ‘Tensor’ in namespace ‘at’ does not name a type
   55 | at::Tensor convert_weight_packed(at::Tensor& weight);
      |     ^~~~~~
`